<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cmd8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDelete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd0">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdEnter">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmd1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOut">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>231</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblError">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Screen Title</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnits">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblKeyPadType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>KeyboardType</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancelKeypad">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
